#60 notification service ui

mariam00001 opened the pull request:
i changes the notification service and the meassege toaster

Reem-A-Hikal committed:
[Admin] NGRX Implementation Action & State #53

Reem-A-Hikal committed:
[Admin] Category Table #52
feat: add categories route and menu item; refactor routes for better organization

- Added a new menu item for 'categories' in the shared data menu.
- Introduced a new route for 'categories' in content.routes.ts, linking to the appropriate module.
- Updated full.routes.ts to include the 'categories' path.
- Refactored existing routes for consistency and readability.
- Renamed user registration request model from 'userRegisterRequest' to 'registerUserRequest' for clarity.

Reem-A-Hikal committed:
[Admin] NGRX Implementation Action & State #53
Refactor category state management: add pageSize to state model and update actions

Reem-A-Hikal committed:
Revert "[Admin] Category Table #52"

This reverts commit 18b0556ddb840b4592ba46a2bde48676a2ef8ab1.

mariam00001 committed:
#55

fix #55 - [Admin] Make Delete

mariam00001 committed:
#60

fix #60 [Service] Notification Service UI

MhmedRady committed:
Create ci.yml

MhmedRady committed:
Merge branch 'main' into develop

MhmedRady committed:
fix admin panel

MhmedRady committed:
change Server dir

MhmedRady committed:
Delete ci.yml

AthrAlmarefa committed:
remove work yml (#70)

Co-authored-by: MohmedRady <75885710+MhmedRady@users.noreply.github.com>

MhmedRady committed:
Create ci.yml

MhmedRady committed:
remove github

MhmedRady committed:
add github

MhmedRady committed:
add Client Mobile dir

AthrAlmarefa committed:
Merge branch 'develop' into service-notification-service-ui

MhmedRady committed:
Update ci.yml

MhmedRady committed:
Update ci.yml

MhmedRady committed:
Update ci.yml

mariam00001 committed:
#67 Service notification service UI

fix #67 Service notification service UI
i worked in notification service and message ts of toaster and used meassege template to test my code

MhmedRady committed:
Merge branch 'develop' into service-notification-service-ui

MhmedRady committed:
Update build.yml

mariam00001 committed:
#60 notification service ui

i changes the notification service and the meassege toaster

mariam00001 committed:
#60 notification service ui

fix #60 notification service ui
change message toaster ts and notification service

mariam00001 committed:
Merge branch 'develop' into notification-service

mariam00001 committed:
#60 notification service ui

#60 notification service ui

MhmedRady commented on Client/Client.Admin/.angular/cache/19.2.19/cuba-tailwind-angular/.tsbuildinfo:
remove this file

MhmedRady commented on Client/Client.Admin/src/app/shared/services/notification.service.ts:
```diff
@@ -1,15 +1,22 @@
-// import {inject, Injectable} from "@angular/core";
```
remove comments

mariam00001 committed:
#60 notification service ui

fix #60 notification service ui

mariam00001 committed:
Merge branch 'develop' into service-notification-service-ui

MhmedRady commented on Client/Client.Admin/src/app/shared/services/notification.service.ts:
Please add a message parameter to every alert and use the Translate Service to handle the message translation

mariam00001 committed:
#60 notification service ui

fix #60 notification service ui
add a message parameter to every alert and use the Translate Service to handle the message translation

mariam00001 committed:
#60 notification service ui

fix #60 notification service ui
add message translate

mariam00001 committed:
Merge branch 'service-notification-service-ui' into notification-service

mariam00001 committed:
Merge branch 'Client]-Service-notification-service-UI' into notification-service

mariam00001 committed:
#60 notification service ui

#60 notification service ui
finall
